README.rst

```diff
@@ -75,26 +75,23 @@ we'd love to help!
 Documentation
 =============
 
-The PyMedPhys documentation is split into four categories:
+The PyMedPhys documentation is split into five categories:
 
 1. `App Users Guide`_: for those who only wish to use ready-made PyMedPhys
-   tools
-
-.. _`App Users Guide`: https://docs.pympedphys.com/appusers/index.rst
+   GUI tools.
```

Comment:
I used the word "GUI" before... But I wonder if the target audience for that part of the docs won't know what that acronym is?

Reply:
Fair. I'm sure just "tools" is fine.